qemu/tests/qemu_disk_img_convert.py

```diff
@@ -80,13 +81,20 @@ def run(test, params, env):
     if not md5:
         test.error("Fail to save tmp file")
     convert_test.destroy_vm()
-    n_params = convert_test.convert()
-    convert_test.compare_test(n_params)
-    convert_test.verify_info(n_params)
-    convert_test.start_vm(n_params)
 
-    # check md5sum after conversion
-    ret = convert_test.check_file(t_file, md5)
-    if not ret:
-        test.error("image content changed after convert")
-    convert_test.clean()
+    cluster_size_set = params.get("cluster_size_set")
+    if cluster_size_set == None:
+        cluster_size_set = ""
+    for cluster_size in re.split(r"\s+", cluster_size_set.strip()):
```

Comment:
Why use `re.split(r"\s+", cluster_size_set.strip())` ?
You could get a list by `cluster_size_set.split()`.
```
Pls refer to https://docs.python.org/3/library/stdtypes.html?highlight=split#str.split

str.split(sep=None, maxsplit=-1)¶

    Return a list of the words in the string, using sep as the delimiter string. If maxsplit is given, at most maxsplit splits are done (thus, the list will have at most maxsplit+1 elements). If maxsplit is not specified or -1, then there is no limit on the number of splits (all possible splits are made).

    If sep is given, consecutive delimiters are not grouped together and are deemed to delimit empty strings (for example, '1,,2'.split(',') returns ['1', '', '2']). The sep argument may consist of multiple characters (for example, '1<>2<>3'.split('<>') returns ['1', '2', '3']). Splitting an empty string with a specified separator returns [''].

    For example:
    >>>

    >>> '1,2,3'.split(',')
    ['1', '2', '3']
    >>> '1,2,3'.split(',', maxsplit=1)
    ['1', '2,3']
    >>> '1,2,,3,'.split(',')
    ['1', '2', '', '3', '']

    If sep is not specified or is None, a different splitting algorithm is applied: runs of consecutive whitespace are regarded as a single separator, and the result will contain no empty strings at the start or end if the string has leading or trailing whitespace. Consequently, splitting an empty string or a string consisting of just whitespace with a None separator returns [].

    For example:
    >>>

    >>> '1 2 3'.split()
    ['1', '2', '3']
    >>> '1 2 3'.split(maxsplit=1)
    ['1', '2 3']
    >>> '   1   2   3   '.split()
    ['1', '2', '3']




```